src/containers/CategoryPage/CategoryPage.tsx

```diff
@@ -289,44 +289,50 @@ const CategoryPage = () => {
               <Grid item xs={4} className="large-screen">
                 <Divider />
                 <div className="title">Preferred</div>
-                {content.data.preferred.map(({ name, enabled }) => (
-                  <CategoryListItem
-                    key={name}
-                    hovered={hovered === name}
-                    category={category}
-                    techName={name}
-                    enabled={enabled}
-                    onClick={() => setTechnology(name)}
-                  />
-                ))}
+                {[...content.data.preferred]
```

Comment:
Do we really need array destructing here?
```suggestion
                {content.data.preferred
```

Reply:
The `content.data.*` array is used in both the <CategoryListItem> and <CategoryRadar> but they were in the reverse order. The use of destructuring is to make a copy of the array and ensure the reverse() only applied on one side. In terms of performance, it's definitely not a pretty solution. However, the site is static generated so performance impact is negligible.

Reply:
makes sense